namespaces/init.go

```diff
@@ -82,6 +82,12 @@ func Init(container *libcontainer.Config, uncleanRootfs, consolePath string, pip
 		}
 	}
 
+	cloneFlags := GetNamespaceFlags(container.Namespaces, true)
```

Comment:
I think maybe better to do all this checks before real configuration? Maybe create simple function with all this checks.